common/src/main/java/me/shedaniel/clothconfig2/gui/widget/DynamicEntryListWidget.java

```diff
@@ -81,6 +83,21 @@ public DynamicEntryListWidget(Minecraft client, int width, int height, int top,
         this.backgroundLocation = backgroundLocation;
     }
     
+    /**
+     * Get all visible children. I.e. hidden config entries are filtered out.
+     * 
+     * @return an unmodifiable {@link List} of visible entries
+     */
+    public List<E> visibleChildren() {
+        return this.children().stream()
+                .filter(entry -> {
+                    if (entry instanceof AbstractConfigEntry<?> configEntry)
```

Comment:
Since this list widget is used across Cloth Config and also for other purposes, it would be nice if this does not reference the actual config implementations.
I would extract this to another interface, or add a isDisplayed / isVisible method to the existing Entry class.

Reply:
Introduced `HidableWidget` and `DisableableWidget`. Moved the implementations up to `DynamicEntryListWidget.Entry`.